usaspending_api/accounts/models.py

```diff
@@ -3,11 +3,33 @@
 from usaspending_api.common.models import DataSourceTrackedModel
 
 
-# Table #3 - Treasury Appropriation Accounts.
+class FederalAccount(models.Model):
+    """
+    Represents a single federal account. A federal account encompasses
+    multiple Treasury Account Symbols (TAS), represented by
+    :model:`accounts.TreasuryAppropriationAccount`.
+    """
+    agency_identifier = models.CharField(max_length=3, db_index=True)
+    main_account_code = models.CharField(max_length=4, db_index=True)
+    account_title = account_title = models.CharField(max_length=300)
+
+    class Meta:
+        managed = True
+        db_table = 'federal_account'
+        unique_together = ('agency_identifier', 'main_account_code')
+
+
 class TreasuryAppropriationAccount(DataSourceTrackedModel):
+    """Represents a single Treasury Account Symbol (TAS)."""
     treasury_account_identifier = models.AutoField(primary_key=True)
+    federal_account = models.ForeignKey('FederalAccount', models.DO_NOTHING, null=True)
     tas_rendering_label = models.CharField(max_length=50, blank=True, null=True)
     allocation_transfer_agency_id = models.CharField(max_length=3, blank=True, null=True)
+    # todo: update the agency details to match FederalAccounts. Is there a way that we can
+    # retain the text-based agency TAS components (since those are attributes of TAS
+    # while still having a convenient FK that links to our agency tables using Django's
+    # default fk naming standard? Something like agency_identifier for the 3 digit TAS
+    # component and agency_id for the FK?)
```

Comment:
I'm pretty sure the answer is "yes, why not?"  It's a small offense against the principles of normalization, but since this isn't an OLTP system, sacrificing that for performance is fine.